sources/web/datalab/polymer/components/datalab-files/datalab-files.html

```diff
@@ -174,10 +174,13 @@
         <iron-icon icon="add-box"></iron-icon>
         <span>Folder</span>
       </paper-button>
-      <paper-button class="toolbar-button">
+      <paper-button class="toolbar-button" on-click="_altUpload">
```

Comment:
The label can't be changed easily unfortunately, since it's a browser implementation. The only way I found to change it is to write our own open dialog.